safehttp/incoming_request.go

```diff
@@ -32,7 +32,9 @@ type IncomingRequest struct {
 	URL       *url.URL
 }
 
-func newIncomingRequest(req *http.Request) *IncomingRequest {
+// NewIncomingRequest creates an safehttp.IncomingRequest
```

Comment:
nit: creates an IncomingRequest

Don't prefix with the package name if the symbol is in the same package.